### PR TITLE
Feature/pegout status service

### PR DESCRIPTION
**Pull request by ronaldsg20**

### CODE REVIEW

File: src/dependency-injection-handler.ts [hunk 1]
```diff
@@ -133,10 +132,15 @@ export class DependencyInjectionHandler {
...
       .inScope(BindingScope.SINGLETON);
 
-      app
-      .bind(ServicesBindings.RSK_BLOCK_PROCESSOR_PUBLISHER)
-      .toClass(NodeBridgeDataProvider)
-      .inScope(BindingScope.SINGLETON);
+    app
+    .bind(ServicesBindings.RSK_BLOCK_PROCESSOR_PUBLISHER)
+    .toClass(NodeBridgeDataProvider)
+    .inScope(BindingScope.SINGLETON);
```
Comment (josedahlquist) [inline src/dependency-injection-handler.ts:138]:
```suggestion
    app
      .bind(ServicesBindings.RSK_BLOCK_PROCESSOR_PUBLISHER)
      .toClass(NodeBridgeDataProvider)
      .inScope(BindingScope.SINGLETON);
```

File: src/models/rsk/pegout-status-data-model.ts [hunk 2]
```diff
@@ -19,6 +20,58 @@ export class PegoutStatusDataModel implements SearchableModel {
...
   status: PegoutStatus;
   btcRawTransaction: string;
+}
+
+export class PegoutStatusAppDataModel implements PegoutStatusDataModel{
+  constructor(data?: Partial<PegoutStatusAppDataModel>) {
+    Object.assign(this, data);
+  }
+
+  static fromPegoutStatusDbDataModel(model: PegoutStatusDbDataModel):PegoutStatusAppDataModel {
```
Comment (josedahlquist) [inline src/models/rsk/pegout-status-data-model.ts:30]: does it need to be from `PegoutStatusDbDataModel` implementation or could it just be from `PegoutStatusDataModel` interface?

File: src/services/pegout-status/pegout-status.service.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import {PegoutStatus, PegoutStatusAppDataModel} from "../../models/rsk/pegout-status-data-model";
+import {PegoutStatusMongoDbDataService} from "../pegout-status-data-services/pegout-status-mongo.service";
+
+export class PegoutStatusService {
+    private logger: Logger;
+    private pegoutStatusDataService: PegoutStatusMongoDbDataService;
+
+    constructor(
+        @inject(ServicesBindings.PEGOUT_STATUS_DATA_SERVICE)
+            pegoutStatusDataService: PegoutStatusMongoDbDataService,
```
Comment (josedahlquist) [inline src/services/pegout-status/pegout-status.service.ts:13]: We should use the interface everytime we can
```suggestion
            pegoutStatusDataService: PegoutStatusDataService,
```

**Review by josedahlquist — CHANGES_REQUESTED** [review]

**Review by alexjavabraz — APPROVED** [review]
SGTM

**Pull request merged**